botocore/signers.py

```diff
@@ -196,6 +208,65 @@ def generate_presigned_url(self, request_dict, expires_in=3600,
         return request.url
 
 
+class CloudFrontSigner(object):
+
+    @staticmethod
```

Comment:
Call it something else please to make it clear this isn't just a basic base64 encoding.  Looks like it's an encoding that also makes it valid in a url.

I'd also be in favor of removing the staticmethod and just making it internal.